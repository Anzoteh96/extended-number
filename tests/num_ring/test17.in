534949699880679787328923195739152005717973008095768
455970431397171445097714255529546049470034613061305
-391750193670380902203019650972282066568396553004288
