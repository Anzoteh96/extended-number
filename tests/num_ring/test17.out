Enter the number base b:
Enter two numbers n and m:
b is 534949699880679787328923195739152005717973008095768
n mod b is 455970431397171445097714255529546049470034613061305
m mod b is 143199506210298885125903544766869939149576455091480
n + m is 64220237726790542894694604557263982901638060057017
n - m is 312770925186872559971810710762676110320458157969825
n * m is 212743639228108039177471937205661572646669967327624
n ** m is 127557812984710768001014793798583104630704538813561
